### PR TITLE
Use constants macros

### PR DESCRIPTION
**Pull request by kylex2**
This was made using scripting and I'm not sure I have checked all lines manually yet. I avoided adv7511 for now since it doesn't use types from stdint.h. I create this pull request to let the tests run.

### CODE REVIEW

File: drivers/adc/ad9083/ad9083_api/adi_ad9083_rx.c [hunk 1]
```diff
@@ -1209,4 +1209,4 @@ int32_t adi_ad9083_rx_datapath_config_set(adi_ad9083_device_t *device,
   return API_CMS_ERROR_OK;
 }
 
-/*! @} */
```
Comment (amiclaus) [inline drivers/adc/ad9083/ad9083_api/adi_ad9083_rx.c:1212]: nit: unrelated change.

**Review by amiclaus — COMMENTED** [review]
one nitpick inline + some code style issues (CI detected) + commit body is missing for most of the commits.

On the other hand, is there any strong reason to do these changes?

**Review by adrimbarean — COMMENTED** [review]
Looks good to me. Just need to apply astyle to the changes.
